uktrade/data-hub-frontend
# CLS2-905 Create EYB lead details page

## Description of change

This adds an EYB lead details page to Data Hub.

The e2e tests had been failing due to the connection to the api timing out, so this PR also extends the timeout duration from 3 to 10 minutes.

## Test instructions
**In the sandbox environment**:
Visit http://localhost:3000/investments/eyb-leads/c836ca84-d9d1-4dcc-980f-40567be1d357/details to view the page

**To test it locally**:
Populate the database with EYB Lead instances:

- In Docker, go to the api container
- Go to the 'Exec' tab
- Run the following commands

```
python manage.py shell
from datahub.investment_lead.test.factories import EYBLeadFactory

EYBLeadFactory.create_batch(10)
```
In Django Admin, go to **Investment_Lead>Eyb leads** and select one of the EYB leads uuids.
Visit the frontend details page for EYB lead by modifying the following url:
thathttp://localhost:3000/investments/eyb-leads/<selected uuid>/details

The page should display as below.
The company name in the table should be clickable and take you to the relevant company page.
The company website in the table should be clickable and take you to the relevant website.

<img width="512" alt="Screenshot 2024-09-30 at 14 51 04" src="https://github.com/user-attachments/assets/00d28454-d2a5-4804-94cb-b6e6ad871e53">

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
